composer.json

```diff
@@ -149,11 +150,10 @@
         "phpstan/extension-installer": "^1.4",
         "phpstan/phpstan": "^2.1",
         "phpstan/phpstan-phpunit": "^2.0",
-        "phpunit/php-code-coverage": "^11.0",
```

Comment:
did you mean to remove this one?

Reply:
Nope, looks like it got cut out with the rebase.  Will put it back.